dev/app-shell/elements/arc-footer.html

```diff
@@ -57,9 +60,55 @@
     }
     _onPlanSelected(e, suggestion) {
       this._fire('suggest', suggestion);
+      this._doBackendSearch(null);
+    }
+    _onSearchClick(e) {
+      this._updateSearchProps('*');
+    }
+    _onSearchChange(e) {
+      let search = e.target.value.toLowerCase();
+      if (!search || search == '*' || search[search.length - 1] == ' ') {
+        this._doBackendSearch(search);
+      } else {
+        this._updateSearchProps(search);
+      }
+    }
+    _onSeachDone(e) {
+      this._doBackendSearch(e.target.value.toLowerCase());
+    }
+    _doBackendSearch(search) {
+      this._fire('replan', {search: (search == '*' ? null : search)});  // trigger planner
```

Comment:
This event is providing a `search` string, so suggest it should be named something like `search` instead of `replan`.

Reply:
done